### PR TITLE
Added more projects

### LINKED ISSUE #46: Add more project
If anyone intersted to take this issue .. Comment on this 
.
Task : You have to add some project in the Project_ideas section . 
Thank u :smile:

### PR DESCRIPTION
**Pull request by AnmolSirola**
# Descripion📜
Added some new projects in the project section

Fixes #46

# Type of Change 📝

- [x] Improve the readme of any project

# Checklist ✔

- [x] I'm SSOC'22 Contributor

### CODE REVIEW

File: Project_Ideas/README.md [hunk 1]
```diff
@@ -16,3 +16,4 @@
...
 | Crowdfunding | Blockchain-powered crowdfunding is anonymous, safe, and borderless, it has the potential to be more democratic. Blockchain can help to speed up the development of a worldwide crowdfunding platform that anybody in the world may use. It will enable all-encompassing crowdfunding. Instead of giving money to a comapny or a third party , with the help of smart contract the people will be able to directly give money to the people who are asking for funds.
+| DAO | Decentralized Autonomous Organizations they are an evolution of organizations the way we know.People who invest in DAOs receive tokens and have the right to vote on DAO decisions because there is no centralized leadership. It is allows greater transparency, as all actions are recorded on an open code public record, allowing people to examine the history of actions.They can finance projects, communities, provide resources to create a pool of investments, a freelancer network, a charity fund , and so on.
```
Comment (Vikash-8090-Yadav) [inline Project_Ideas/README.md:19]: DAO is not a project its a  term used in the blockchain . like voting application  comes in dao etc

**Comment by AnmolSirola** [comment]
@Vikash-8090-Yadav I have removed one project and added few more plz check it now .

**Comment by Vikash-8090-Yadav** [comment]
Can u please  tell the no of project that u have added ?

**Comment by AnmolSirola** [comment]
@Vikash-8090-Yadav I have added 12 project ideas.

**Comment by Vikash-8090-Yadav** [comment]
Ok

**Pull request merged**